packages/apputils/src/toolbar.tsx

```diff
@@ -440,11 +440,12 @@ export function ToolbarButtonComponent(props: ToolbarButtonComponent.IProps) {
   // We avoid a click event by calling preventDefault in mousedown, and
   // we bind the button action to `mousedown`.
   const handleMouseDown = (event: React.MouseEvent) => {
-    // Fire action only when left button is pressed.
-    if (event.button === 0) {
-      event.preventDefault();
-      props.onClick();
-    }
+    event.preventDefault();
+  };
+
+  const handleClick = (event: React.MouseEvent) => {
+    event.preventDefault();
+    props.onClick();
   };
 
   const handleKeyDown = (event: React.KeyboardEvent) => {
```

Comment:
In theory, now that the button action is bound to the `click` we could get rid of this.
The button is automatically activated if `Enter` or `Space` is pressed while it has focus (obtained by _tabbing_), this is one of the reasons to prefer `click` over `mouseup` to trigger actions.
Unfortunately, I encountered a bug in Firefox on Linux preventing us from doing so (with Chrome it seems impossible to move focus to the buttons by _tabbing_).
`Enter`works as expected, `Space` instead activates the button twice. (can you confirm?)
So we had still to stay with this, at least for the `Space` case.
Anyway, `preventDefault` should be called, otherwise the action is triggered multiple times, explicitly by the handler and as default action of `keydown`.